### DISCUSSION TITLE
GameObject.FindObjectsOfType<SomeScript> 리턴값 배열에 undefined가 들어있습니다.

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#610] [Issue Report (Legacy)]
Zepeto 패키지 버전은 1.1.5이며, 아래 코드와 같이 GameObject.FindObjectsWithTag<SomeScript> 를 사용시
return되는 배열에 undefined만 들어있습니다.

```ts
const results = GameObject.FindObjectsOfType<SomeScript>(false);

for(const result of results) {
  console.log(result);
  // undefined가 찍힘
}
```

**Comment by Cha-SeongHun** [comment]
1.1.7 에서 확인결과
GameObject.FindObjectsWithTag<T>() 함수가 똑같이 동작하지 않는 것을 확인하였습니다.

https://github.com/naverz/zepeto-studio-kor/discussions/463
링크의 이슈 또한 1.1.7에서도 재현되고 있으며 동일한 원인으로 보입니다.

**Reply by Zepeto-TechEvangelist** [reply]
상세한 제보 감사 드립니다.
말씀하신대로 동일한 원인이어서 내부에서 파악 중입니다.
다만 그때도 말씀드렸다시피 시간은 다소 소요되는 점 양해 부탁 드립니다🙏
